classes/class-wc-connect-shipping-label.php

```diff
@@ -394,7 +394,7 @@ public function get_label_payload( $post_order_or_id ) {
 		 * @param  array $item Item to check for shipping.
 		 * @return bool
 		 */
-		protected function filter_items_needing_shipping( $item ) {
```

Comment:
Can you please provide some context on the PR's description to clarify why the method's visibility changed?

Reply:
Hey @frosso -- this might help explain things here:

```
// WooCommerce Services compatibility.
add_filter( 'wc_connect_meta_box_payload', 'wc_services_filter_payload', 10, 4 );

function wc_services_filter_payload( $payload, $args, $order, $label ) {

	if ( isset( $payload[ 'shippingLabelData' ][ 'formData' ][ 'selected_packages' ] ) ) {

		add_filter( 'woocommerce_order_get_items', array( WC_PB()->order, 'get_order_items' ), 20, 2 );
		add_filter( 'woocommerce_order_item_product', array( WC_PB()->order, 'get_product_from_item' ), 20, 2 );

		$payload[ 'shippingLabelData' ][ 'formData' ][ 'selected_packages' ] = $label->get_selected_packages( $order );

		$payload[ 'items' ] = sizeof( array_filter( $order->get_items( 'line_item' ), array( $label, 'filter_items_needing_shipping' ) ) );

		remove_filter( 'woocommerce_order_get_items', array( WC_PB()->order, 'get_order_items' ), 20 );
		remove_filter( 'woocommerce_order_item_product', array( WC_PB()->order, 'get_product_from_item' ), 20 );
	}

	return $payload;
}
```

In short, we are changing the item count due to the presence of assembled product kits, and I thought it might be better to reuse this function that to re-write it on our end.

Reply:
Thanks for the explanation!